jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/mapper/UserTaskInstanceWithCustomVarsQueryMapper.java

```diff
@@ -96,7 +96,8 @@ protected UserTaskInstanceWithVarsDesc buildInstance(DataSet dataSetResult, int
                 getColumnStringValue(dataSetResult, COLUMN_TASK_PROCESSID, index),//processId,
                 getColumnLongValue(dataSetResult, COLUMN_TASK_PROCESSINSTANCEID, index),//processInstanceId,
                 getColumnDateValue(dataSetResult, COLUMN_CREATEDON, index),//createdOn,
-                getColumnDateValue(dataSetResult, COLUMN_DUEDATE, index)//dueDate
+                getColumnDateValue(dataSetResult, COLUMN_DUEDATE, index),//dueDate
+                getColumnDateValue(dataSetResult, COLUMN_TASK_SLA_DUE_DATE, index)//slaDueDate
```

Comment:
Apart from that column, shouldn't we add the rest of missing ones?

```
getColumnStringValue(dataSetResult, COLUMN_SUBJECT, index),
getColumnStringValue(dataSetResult, COLUMN_CORRELATIONKEY, index), 
getColumnIntValue(dataSetResult, COLUMN_PROCESSTYPE, index),
getColumnIntValue(dataSetResult, COLUMN_TASK_SLA_COMPLIANCE, index)
```

@VaniHaripriya @fjtirado wdyt?